torch_geometric/distributed/dist_loader.py

```diff
@@ -0,0 +1,103 @@
+import atexit
+import os
+from typing import Dict, List, Optional, Union
+
+import torch
+import torch.multiprocessing as mp
+
+from .dist_context import DistContext, DistRole
+from .dist_neighbor_sampler import DistNeighborSampler, close_sampler
+from .rpc import global_barrier, init_rpc
+
+
+class DistLoader():
+    def __init__(
+            self,
+            neighbor_sampler: DistNeighborSampler,
+            current_ctx: DistContext,
+            rpc_worker_names: Dict[DistRole, List[str]],
+            master_addr: str,
+            master_port: Union[int, str],
+            # Optional[Union[ChannelBase, mp.Queue()]],
+            channel: mp.Queue(),
+            num_rpc_threads: Optional[int] = 16,
+            rpc_timeout: Optional[int] = 180,
+            device: Optional[torch.device] = torch.device('cpu'),
```

Comment:
```suggestion
            device: Optional[torch.device] = None,
```
would prefer `None` as this is more common in PyTorch.

Reply:
fixed